crates/config/src/sections/passwords.rs

```diff
@@ -35,6 +35,10 @@ fn default_enabled() -> bool {
     true
 }
 
+fn default_minimum_complexity() -> u8 {
+    3
```

Comment:
This might be too disruptive to existing deployments? Although it's a good default, so we might just have to make a big warning in the upgrade/release notes I guess 🤷

Reply:
It's not like it will stop anyone from logging in, so I'm not sure I share the concern to be honest. But sure, a warning in the release notes is always the right way to do it